pkg/minikube/node/start.go

```diff
@@ -366,6 +366,12 @@ func startHost(api libmachine.API, cc *config.ClusterConfig, n *config.Node) (*h
 		return host, exists, err
 	}
 
+	// don't try to re-create if cpu count is not enough
```

Comment:
Instead of lines 364-374, if we include the `FailFastError` from above we could replace these lines with:

```go
	if _, ff := err.(FailFastError); ff {
		glog.Infof("will skip retrying to create machine because error is not retriable: %v", err)
		return host, exists, err
	}

```